llarp/handlers/tun.cpp

```diff
@@ -380,12 +380,54 @@ namespace llarp
     {
       // llarp::LogInfo("Tun.HandleHookedDNSMessage ", msg.questions[0].qname, "
       // of type", msg.questions[0].qtype);
+      std::string qname;
+      if(msg.answers.size() > 0)
+      {
+        const auto &answer = msg.answers[0];
+        if(answer.HasCNameForTLD(".loki"))
```

Comment:
Shouldn't this have a `and not (has A record)`?

Reply:
it will strip any A records out